api/src/main/java/io/opentelemetry/common/Attributes.java

```diff
@@ -79,7 +83,7 @@ public static Attributes empty() {
   }
 
   /** Returns a {@link Attributes} instance with a single key-value pair. */
-  public static <T> Attributes of(AttributeKey<T> key, T value) {
+  public static <T> Attributes of(AttributeKey<T> key, @Nonnull T value) {
```

Comment:
I don't think we use `@Nonnull`, we default to non-null. We're missing package-info.java to set ParametersNonNullByDefault but that's a better fix I think

Reply:
I agree. As I was writing this, I was also wondering why we didn't put the same annotation on the key. :)

Having it set for the package as a whole is *much* better solution. I'll make it so.

Reply:
done